docs/project/contribution_tools.md

```diff
@@ -255,6 +254,12 @@ work reliably include:
 
 ## Troubleshooting build issues
 
+### `bazel clean`
+
+Changes to packages installed on your system may not be noticed by `bazel`. This
+includes things such as changing LLVM versions, or installing libc++. Running
+`bazel clean` should force cached state to be rebuilt.
```

Comment:
Can we leave a breadcrumb of the exact error here, so a search will turn it up as well?

Suggestion:

As one example, Bazel may raise "absolute path inclusion(s) found in rule" and "includes the following non-builtin files with absolute paths" as an error for libc++ includes if the location of the standard library header files changes.

Reply:
I'm now tackling the libc++ issue down a different route, just validation during repo setup (also clang versions that we'd discussed in the past)